packages/cli/src/commands/list.ts

```diff
@@ -31,6 +34,7 @@ const help = () => {
     'DIR'
   )}    Path to the global ${'`.vercel`'} directory
     -d, --debug                    Debug mode [off]
+    -y, --yes                      Skip the confirmation prompt
```

Comment:
This should be `--confirm` because its not asking a yes or no question. See `vc link --confirm` and `vc dev --confirm` for existing usage.